pages/index.js

```diff
@@ -56,6 +56,15 @@ export default function Home({
     }));
   };
 
+  const clearSituation = () => {
+    fetch(`${process.env.NEXT_PUBLIC_BASE_URL}/api/situation`, {
+      method: "delete",
+    });
+    document.getElementById("location-select").value = "";
+    document.getElementById("age").value = "";
+    document.getElementById("income-select").value = "";
```

Comment:
It is probably worth setting situation in state for consistency instead

Reply:
Just calling `setSituation` doesn't clear the fields, so these are required.

Reply:
That is weird. It should be able to update based off of state. Something to look into in the future perhaps